File: 867-transpose-matrix/867-transpose-matrix.java
class Solution {
    public int[][] transpose(int[][] m) {
        int[][] res = new int[m[0].length][m.length];
        for(int i = 0;i<m[0].length;i++){
            for(int j = 0;j<m.length;j++){
                res[i][j] = m[j][i];
            }
        }
        return res;
    }
}

File: 867-transpose-matrix/867-transpose-matrix.cpp
class Solution {
public:
    vector<vector<int>> transpose(vector<vector<int>>& v) {
        vector<vector<int>>res(v[0].size(),vector<int>(v.size(),0));
        for(int i = 0;i<v[0].size();i++){
            for(int j = 0;j<v.size();j++){
                res[i][j] = v[j][i];
            }
        }
        return res;
    }
};